apps/passport-server/src/services/telegramService.ts

```diff
@@ -0,0 +1,125 @@
+import { EdDSATicketPCD, EdDSATicketPCDPackage } from "@pcd/eddsa-ticket-pcd";
+import { Bot, InlineKeyboard } from "grammy";
+import { fetchTelegramConversation } from "../database/queries/telegram/fetchTelegramConversation";
+import { insertTelegramConversation } from "../database/queries/telegram/insertTelegramConversation";
+import { markTelegramUserAsVerified } from "../database/queries/telegram/updateTelegramConversation";
+import { ApplicationContext } from "../types";
+import { logger } from "../util/logger";
+
+export class TelegramService {
+  private context: ApplicationContext;
+  private bot: Bot;
+  private privateChatId: string;
```

Comment:
this should probably a map from event to private chat id, so that we can support many events using with this service